CommitsOfFury/src/CommitsOfFury/Class2.cs

```diff
@@ -9,7 +9,7 @@ public class Class2 : IAmAClass
     {
         public string TheThingToSay()
         {
-            return "Hello";
+            return "Hello everyone";
```

Comment:
This implementation doesn't support localization. Please see Lori's upcoming session!